runtime/runtime-params-estimator/src/main.rs

```diff
@@ -79,5 +93,100 @@ fn main() {
     let str = serde_json::to_string_pretty(&runtime_config)
         .expect("Failed serializing the runtime config");
     fs::write(state_dump_path.join("runtime_config.json"), &str)
-        .unwrap_or_else(|err| panic!("Failed to write runtime config to file {}", err))
+        .context("Failed to write runtime config to file")?;
+    Ok(())
+}
+
+fn main_docker(state_dump_path: &Path) -> anyhow::Result<()> {
+    exec("docker --version").context("please install `docker`")?;
+
+    let project_root = project_root();
+    if exec("docker images -q rust-emu")?.is_empty() {
+        // Build docked image if there isn't one already.
+        let status = Command::new("docker")
+            .args(&["build", "--tag", "rust-emu"])
+            .arg(project_root.join("runtime/runtime-params-estimator/emu-cost"))
+            .status()?;
+        if !status.success() {
+            anyhow::bail!("failed to build a docker image")
+        }
+    }
+
+    let init = {
+        // Build a bash script to run inside the container. Concatenating a bash
+        // script from strings is fragile, but I don't know a better way.
+
+        let mut buf = String::new();
+        buf.push_str("set -ex;\n");
+        buf.push_str(
+            "\
+cargo build --manifest-path /nearcore/Cargo.toml \
```

Comment:
Can we mount directory to `/host/nearcore` instead of `/nearcore`?
The minor issue with it is that we use `/host/nearcore` in other places, and it would be convenient to use the same folder everywhere:
- https://github.com/near/nearcore/blob/master/runtime/runtime-params-estimator/emu-cost/README.md
- https://github.com/near/nearcore/blob/master/runtime/runtime-params-estimator/src/function_call.rs#L61